DfE.Hexagonal.Driver.Application.Tests/Tests/School/GetAllSchoolTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using DfE.Hexagonal.Application.UseCases.School.GetAllSchools;
+using DfE.Hexagonal.Driver.Application.Tests.TestDoubles;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DfE.Hexagonal.Driver.Application.Tests.Tests.School
+{
+    public class GetAllSchoolTests
+    {
+        [Fact]
+        public async void GetAllSchools_returns_a_list_of_schools()
+        {
+            // arrange
+
+            var repository = SchoolRepositoryTestDoubles.MockFor(new List<SchoolOutput>()
+            {
+                new SchoolOutput()
+                {
+                    Id = Guid.NewGuid(),
+                    Name = "test",
+                    Description = "test",
+                    OpenDate = DateTime.Now
+                },
+                new SchoolOutput()
+                {
+                    Id = Guid.NewGuid(),
+                    Name = "test2",
+                    Description = "test2",
+                    OpenDate = DateTime.Now
+                }
+            });
```

Comment:
Perhaps we could extract this out to a variable, and then remove the magic number from the assertion below?

e.g., `Assert.Equal(inputSchools.Count(), result.Schools.Count());`